pkg/cronjob/spark_dependencies.go

```diff
@@ -113,6 +115,10 @@ func getStorageEnvs(s v1.JaegerStorageSpec) []corev1.EnvVar {
 			{Name: "ES_PASSWORD", Value: sFlagsMap["es.password"]},
 			{Name: "ES_CLIENT_NODE_ONLY", Value: strconv.FormatBool(s.Dependencies.ElasticsearchClientNodeOnly)},
 			{Name: "ES_NODES_WAN_ONLY", Value: strconv.FormatBool(s.Dependencies.ElasticsearchNodesWanOnly)},
+			{Name: "ES_TLS", Value: sFlagsMap["es.tls"]},
```

Comment:
This would not enable TLS in the spark dependencies https://github.com/jaegertracing/jaeger-operator/issues/294. 

Remove these for now.